src/diffusers/pipelines/stable_diffusion_3/pipeline_stable_diffusion_3.py

```diff
@@ -224,7 +225,7 @@ def _get_t5_prompt_embeds(
         text_inputs = self.tokenizer_3(
             prompt,
             padding="max_length",
-            max_length=self.tokenizer_max_length,
```

Comment:
Won't this change lead to the generation of different results for a prompt that had more than `self.tokenizer_max_length`? Previously it was 77 and now it is 256. So, I think it will lead to different results in that case. 

I am okay with this because this will likely improve the results but we should emphasize it in our release notes. WDYT? Cc: @yiyixuxu

Reply:
yeah agree

Reply:
As additional information about this, I think I mentioned before, but there's a lot of people testing SD3 with the hippo example from the paper and that prompt is approx. 160 tokens, more than the double of the current default we have.